### ISSUE TITLE
Shell script config disappear after updating other config.

### ISSUE THREAD

**Issue by Sujin-Kim1** [lablup/backend.ai-webui#1231]
**Describe the bug**
When updating other config and refreshing the page, the shell script config menu disappear.

**To Reproduce**
1. Go to 'usersettings' page
2. Update other configs such as system language, mini-ui etc.
3. Refresh

**Expected behavior**
Always visible if API support editing shell script.

**Screenshots**
![image](https://user-images.githubusercontent.com/28584164/154218974-59538997-a144-47fc-a554-632ff2601cab.png)

**Client:**
 - OS: macOS 
 - Browser version: Chrome, Safari

**Additional context**
This error occurs 100% in the cloud. And if you want to make visible the script config menu, you have to close the current tab and re-open it.
But in the development environment, it occurs sometimes and it can be visible if you refresh the page many times.

**Comment by studioego** [comment]
Today, I registered the cloud Backend.AI service.

I also found the issue that when updating other config and refreshing the page, the shell script config menu disappear.

the issue appear the error on the browser.  

<img width="1464" alt="image" src="https://user-images.githubusercontent.com/20002/180716736-125b3e82-4023-4f21-a3b7-a11484dde306.png">

@Sujin-Kim1 , I found that 'isAPIVersionCompatibleWith' is missing on the page.
